components/services/markdown/src/index.js

```diff
@@ -28,6 +28,14 @@ class ServiceMarkdown extends Component {
     )
   }
 
+  componentDidUpdate() {
+    const id = document.location.hash.substring(1)
```

Comment:
do you really want to force the id to be passed by as prop? I think this makes the component less reusable, "id" should be a prop passed by to the dumb UI component and the logic behind loading this id should be done in the wrapping component from each vertical

Reply:
We've been talking about this and we're keeping as it is to keep the native spec of HTML with hash and id.

Reply:
We are generating HTML anyways in a web environment in a cycle method that only executes in browser. So it's ok to me to do it here.